CONTRIBUTING.md

```diff
@@ -132,7 +132,23 @@ Note: Tycho always allows references to locally built artifacts, even if they ar
 
 ### Updating the Equinox and JDT dependencies of Tycho
 
-Tycho has Maven dependencies to Equinox and JDT, so these artifact are used from  Maven  Central repository. 
+Tycho has Maven dependencies to Equinox and JDT, so these artifact are used from  Maven  Central repository.
+
+### Profiling the Tycho build
+
+To understand where the build spends most of its time, you can try the following approaches:
+
+#### Add timestamps to Maven logging
+
+You can [add a timestamp to each log line](https://blogs.itemis.com/en/in-a-nutshell-adding-timestamps-to-maven-log-output) produced by Maven. This is the most easy to apply, but you have to do the calculation of the runtime of different goals yourself.
+
+#### Add Maven profiler
+
+Download the [Maven profiler extension](https://github.com/jcgay/maven-profiler) and add it to your aggregator project. It will produce an HTML report for each goal.
+
```

Comment:
Maybe we can describe here the "use extension way" from https://github.com/jcgay/maven-profiler#maven--33x ? This seems most suitable for tycho-builds.

Reply:
I think we should recommend the extension way instead of download/manual install the plugin into maven.

Reply:
I've incorporated all suggestions and described especially the YouMonitor usage as good as I remember. :) I used it last time maybe 2 years ago.